common/autoware_motion_utils/src/marker/marker_helper.cpp

```diff
@@ -85,6 +85,42 @@ inline visualization_msgs::msg::MarkerArray createDeletedVirtualWallMarkerArray(
 
   return marker_array;
 }
+
+inline visualization_msgs::msg::MarkerArray createIntendedPassVirtualMarkerArray(
+  const geometry_msgs::msg::Pose & vehicle_front_pose, const std::string & module_name,
+  const std::string & ns_prefix, const rclcpp::Time & now, const int32_t id,
+  const std_msgs::msg::ColorRGBA & color)
+{
+  visualization_msgs::msg::MarkerArray marker_array;
+
+  // Virtual Wall
+  {
+    auto marker = createDefaultMarker(
+      "map", now, ns_prefix + "direction", id, visualization_msgs::msg::Marker::ARROW,
+      createMarkerScale(2.5 /*length*/, 0.15 /*width*/, 1.0 /*height*/), color);
+
+    marker.pose = vehicle_front_pose;
+    // marker.pose.position.z += 1.0;
```

Comment:
```suggestion
```

Reply:
fixed in 0dd5fe18ed43f624abcc90ca0a79de2752598f75